run

```diff
@@ -0,0 +1,197 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# Define the versions of the two docker images
+django_image="ubuntudesign/django-app:v1.1.0"
+yarn_image="ubuntudesign/yarn:v0.1.0"
+
+# Usage instructions
+USAGE="Usage
+===
+
+  $ ./run \\
+    [-p|--port NUM]  # Specify local port for server \\
+    [--no-debug]     # Turn off Django's DEBUG setting \\
+    [COMMAND]        # Optionally provide a command to run
+
+If no COMMAND is provided, \`start\` will be run.
+
+Commands
+---
+
+- start [-w|--watch]: Run the development server (optionally watching for Sass changes) \\
+- watch: Run \`yarn run watch\`
+- build: Run \`yarn run build\`
+- test: Run \`yarn run test\` and \`./manage.py test\`
+- shell: Open a Django shell
+- node <args>: Run node
+- yarn <args>: Run yarn
+- django <args>: Run ./manage.py <args>
+- clean: Remove all images and containers, any installed dependencies and the .docker-project file
+
+Persistent options
+---
+
+The options can also be defined in a local \`.env\` file, e.g.:
+
+  # These are the default settings
+  PORT=8000                 # Run the server on local port 8000
+  DJANGO_DEBUG=true         # Enable Django's DEBUG mode
+"
+
+##
+# Check docker is installed correctly
+##
+if ! command -v docker >/dev/null 2>&1; then
+    echo "
+    Error: Docker not installed
+    ==
+    Please install Docker before continuing:
+    https://www.docker.com/products/docker
+    "
+    exit 1
+fi
+if grep -q '^docker:' /etc/group && ! groups | grep -q '\bdocker\b'; then
+    echo "
+    Error: `whoami` not in docker group
+    ===
+    Please add this user to the docker group, e.g. with:
+    \$ newgrp docker
+    "
+    exit 1
+fi
+
+# Generate the project name
+if [[ -f ".docker-project" ]]; then
+  project=$(cat .docker-project)
+else
+  project=$((pwd | md5sum 2> /dev/null || md5 -q -s `pwd`) | cut -c1-8)
+  echo $project > .docker-project
+fi
+
+# Defaults
+PORT=8000
+DJANGO_DEBUG=true
+
+# Read current environment variables
+export $(cat .env | grep -v ^\# | xargs)
+
+# Optional arguments
+# ==
+#  --port {port} - The local port to run the web service on
+#  --no-debug - Turn off Django DEBUG mode
+#  --watch - Toggle "yarn run watch" in the background
+#  --help - Show usage instructions
+invalid() {
+    echo "Command not recognised."
+    echo "$USAGE"
+    exit 1
+}
+
+module_volumes=()
+
+while [[ -n "${1:-}" ]] && [[ "${1:0:1}" == "-" ]]; do
+    key="$1"
+
+    case $key in
+        -p|--port)
+            if [ -z "${2:-}" ]; then invalid; fi
+            PORT="$2"
+            shift
+        ;;
+        -m|--module)
```

Comment:
Yes that's a valid concern. In an earlier iteration I used something move descriptive - `--module-override` I think. But I like `--node-module`. I'll change it to that.